provisioner/src/lib.rs

```diff
@@ -233,6 +254,193 @@ impl MyProvisioner {
         }
     }
 
+    async fn get_prefix(&self, project_name: &str) -> String {
+        //TODO: add userid or something else unique here
+        format!("shuttle-dynamodb-{}-", project_name)
```

Comment:
Need to be 100% sure that users can't overlap these prefixes. (Project id or uid probably.)